test/unit_tests/CMakeLists.txt

```diff
@@ -45,6 +45,11 @@ link_libraries(
   gcov
 )
 
+# TODO: These should be coming from the dependency's CMake module or find_package()
```

Comment:
Let's get this `#TODO: ` in place before we merge.

This solution will be some form of this:
https://cmake.org/cmake/help/v3.2/module/FindBoost.html

I'll play with it and make a PR against this branch.

Reply:
Wouldn't `find_package()` set the paths to whatever version of Boost you have installed globally in your system? I thought that we agreed that we're going to stick to the Boost libraries which are downloaded by the CI scripts for now.

Reply:
No, my understanding is that it uses `$BOOST_ROOT` as a hint to find Boost.

Reply:
Well, if we can provide `find_package()` with some hints that should work :+1: